src/Orc.NuGetExplorer.Example/ProjectManagement/ExampleProjects/ExampleProject.cs

```diff
@@ -29,6 +29,10 @@ public ExampleProject(IFrameworkNameProvider frameworkNameProvider)
 
         public ImmutableList<NuGetFramework> SupportedPlatforms { get; set; }
 
+        public bool CanIgnoreDependencies { get; } = true;
```

Comment:
Maybe we should remove the Can, just name it "IgnoreDependencies" and "SupportSideBySide"